.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use CodeIgniter\CodingStandard\CodeIgniter4;
+use Nexus\CsConfig\Factory;
+use PhpCsFixer\Finder;
+
+$finder = Finder::create()
+    ->files()
+    ->in(__DIR__)
+    ->exclude('build')
+    ->append([__FILE__]);
+
+// Optional rule overrides
+$overrides = [];
+
+$options = [
+    'finder'    => $finder,
+    'cacheFile' => 'build/.php-cs-fixer.cache',
+];
+
+/* Reenable after incremental changes are applied
+return Factory::create(new Standard(), $overrides, $options)->forLibrary(
+    'Library',
+    'Tatter Software',
```

Comment:
Is this portion still relevant?

Reply:
Gah, no. 🤦‍♂️ Good catch. I basically apply my templates and then go through and remove all the LLC stuff. I will rebase this out.